src/goOutline.ts

```diff
@@ -122,7 +122,7 @@ export class GoDocumentSymbolProvider implements vscode.DocumentSymbolProvider {
 				label,
 				this.goKindToCodeKind[decl.type],
 				new vscode.Range(document.positionAt(start), document.positionAt(end)),
-				undefined,
+				document.uri,
```

Comment:
The default value for the `uri` field in the `SymbolInformation` is the current document.
And we are passing blank `containerName`
Both are optional parameters.
We are better off not passing either when creating the `symbolInfo`